enclaver/Cargo.lock

```diff
@@ -960,7 +960,7 @@ checksum = "a26ae43d7bcc3b814de94796a5e736d4029efb0ee900c12e2d54c993ad1a1e07"
 
 [[package]]
 name = "enclaver"
-version = "0.4.1"
+version = "0.4.3-dev"
```

Comment:
Shouldn't we cut `0.4.2` first and then bump to `-dev`?

Reply:
We will. I'm going to tag [dfa6e6f](https://github.com/edgebitio/enclaver/pull/187/commits/dfa6e6f89db26dffa7c39aa06cf314a668f35d6b).